src/components/PermissionsManagement/NewRolePopUp.jsx

```diff
@@ -29,14 +30,23 @@ const CreateNewRolePopup = ({ toggle, addNewRole }) => {
         permissions: permissionsChecked,
         permissionsBackEnd,
       };
-      console.log(newRoleObject);
       await addNewRole(newRoleObject);
       toast.success('Role created successfully');
-
       toggle();
     }
   };
 
+  const handleRoleName = e => {
+    const { value } = e.target;
```

Comment:
It would be great if we can add a small line "Special character/symbols not allowed" or a tooltip when someone trying to add symbol 
![image](https://user-images.githubusercontent.com/82662768/229302132-1818302a-5b3e-4b3c-9122-6383eb11af60.png)